### PR TITLE
TR-772 - Rba support

### PR DESCRIPTION
**Pull request by mvelazc0**

### CODE REVIEW

File: detections/endpoint/malicious_powershell_process___encoded_command.yml [hunk 1]
```diff
@@ -56,3 +56,18 @@ tags:
   - Processes.dest
   - Processes.process_id
   security_domain: endpoint
+  impact: 70
+  confidence: 50
+  risk_score: 35
+  context:
+  - Source: Endpoint
```
Comment (josehelps) [inline detections/endpoint/malicious_powershell_process___encoded_command.yml]: minor thing, all of these under context are meant to be simple string eg: `Source:Endpoint` no space in between.

File: detections/endpoint/suspicious_driver_loaded_path.yml [hunk 2]
```diff
@@ -50,3 +50,19 @@ tags:
...
+  - Source: Endpoint
+  - Stage: Execution, Defense Evasion
+  message: Suspicious driver $ImageLoaded$ on $Computer$
+  observable:
+  - name: Computer
+    type: Endpoint
+    role:
+    - Victim
+  - name: ImageLoaded
+  type: File Name
```
Comment (josehelps) [inline detections/endpoint/suspicious_driver_loaded_path.yml]: this seems spaced incorrectly

**Review by josehelps — CHANGES_REQUESTED** [review]
two minor things that need updating, otherwise it looks great.

File: detections/endpoint/malicious_powershell_process___encoded_command.yml (hunk 1 above)
Comment (mvelazc0) [reply detections/endpoint/malicious_powershell_process___encoded_command.yml]: fixed these across all detections

File: detections/endpoint/suspicious_driver_loaded_path.yml (hunk 2 above)
Comment (mvelazc0) [reply detections/endpoint/suspicious_driver_loaded_path.yml]: this was also fixed

**Review by josehelps — DISMISSED** [review]

**Pull request merged**